bin/fiwareDeviceSimulatorCLI

```diff
@@ -326,12 +327,34 @@ function updateScheduledJobsInGoogleSheet() {
   }
 }
 
+/**
+ * Get .env variables
+ */
+function getEnvVariables(file) {
```

Comment:
As far as I understand this function process the elements of the simulation JSON (starting by the root element in a recursive way) and replace each ${...}  by the corresponding value from the env var in `.env`.

I'd suggest a change name (along with explanatory information):

```suggestion
/**
 * Process the elements of the simulation JSON (starting by the root element in a recursive way) and replace
 * each ${...} token by the corresponding environment variable value (supposedly defined in .env file, otherwise
 * the token is replaced by a blank).
 */
function applyEnvVars(file) {
```

Some alternative to applyEnvVar() could be better, but I don't like getEnvVariables() as it could be interpreted as a function to get the .env file or something like that.

Reply:
I'm not sure

> otherwise the token is replaced by a blank

but I guess this is what to happen when for instance `${FOO}` is used by `FOO` is not defined in .env file.